docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -103,17 +105,35 @@ Preparing For a Release
    This should be a "Feature" with "Release Flocker ${VERSION}" as the title, and it should be assigned to yourself.
    The issue does not need a design, so move the issue to the "Coding" state.
 
-#. Create an environment to do a release in:
+#. Create an environment for the release:
+
+   **Linux**
 
    .. prompt:: bash $,(flocker-0.1.2)$ auto
 
       $ git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
-      # Use system site packages e.g. so that "rpm" can be imported
+      # Make system site packages available for import of non-pip dependencies (e.g. "rpm").
       $ mkvirtualenv -a "flocker-${VERSION}" --system-site-packages "flocker-${VERSION}"
       (flocker-0.1.2)$ pip install --ignore-installed --editable .[dev]
-      (flocker-0.1.2)$ admin/create-release-branch --flocker-version=${VERSION}
-      (flocker-0.1.2)$ admin/update-license
-      (flocker-0.1.2)$ git commit -am "Updated copyright in LICENSE file"
+
+   **OS X**
+
+   .. prompt:: bash $,(flocker-0.1.2)$ auto
+
+      $ git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
+      # Make system site packages available for import of non-pip dependencies (e.g. "rpm").
+      # Use system Python with Homebrew's OpenSSL libraries - see FLOC-3044.
```

Comment:
also `brew install libffi` (or `brew install openssl libffi`).

Link to the client testing docs instead of FLOC-3044 because when that is merged soon (and before this) the docs will explain what is relevant.